README.md

```diff
@@ -1,7 +1,33 @@
 # Large files storage for eclipse-tractusx.github.io
 
-To publish bigger files through github pages or distribute them, github suggests to use gits large file support: [about-large-files-on-github](https://docs.github.com/en/repositories/working-with-files/managing-large-files/about-large-files-on-github).
+To publish bigger files through GitHub pages or distribute them, GitHub suggests to use gits large file support: [about-large-files-on-github](https://docs.github.com/en/repositories/working-with-files/managing-large-files/about-large-files-on-github).
+
+## Naming convention
+
+allowed file names **`DATE_KIT_TITLE_VERSION (e.g., 230808_ECOPASS_VIDEO_v1)`**
+
+- DATE: YYMMDD
+- KIT / COMPONENT: ECOPASS, EDC, BPDM, PORTAL, ...
+- TITLE: VIDEO, MANUAL, ...
+- VERSION: v1, v2, ...
 
 ## How to use
 
-Please read up on how to use ```git lfs```. You can also follow the [guide](https://docs.github.com/en/repositories/working-with-files/managing-large-files/installing-git-large-file-storage) provided by GitHub.
+Please read up on how to use `git lfs`. You can also follow the [guide](https://docs.github.com/en/repositories/working-with-files/managing-large-files/installing-git-large-file-storage) provided by GitHub.
+
+## How to add files
+
+To add files to this repository, you need to install git lfs first. Then you can add files to the repository like this:
```

Comment:
This repo works together with https://github.com/eclipse-tractusx/eclipse-tractusx.github.io
Therefore, I would suggest adding some information on how this works together with the other repository. Is there something to keep in mind? Can the largefiles directly be used? How can they be linked in the Docusaurus?

Maybe we should also add a hint to this largefile repo in the https://github.com/eclipse-tractusx/eclipse-tractusx.github.io repo.

Reply:
Thanks for the idea, added some general information how to use static files within our documentaion. [cae843c](https://github.com/eclipse-tractusx/eclipse-tractusx.github.io.largefiles/pull/7/commits/cae843cda1f87c6dd090dff2cec145b55958920c)
Does this fit for us?

Reply:
Yes, this should make it more straightforward for others, thanks!